app/services/raw_event_reporter.rb

```diff
@@ -2,6 +2,7 @@ class RawEventReporter
   def report!(event)
     measurement = parse_measurement(event.data)
     sensor = Sensor.find_or_create_by(device_name: event.coreid)
+    sensor.caption = 'LOL Sensor'
```

Comment:
What does this class do? Does it register new incoming measurements? If yes, wouldn't this line overwrite the caption for every sensor?

Reply:
Good catch. That's bad.